cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/selector/DefaultStorageSelector.java

```diff
@@ -0,0 +1,27 @@
+package com.linkedin.openhouse.cluster.storage.selector;
```

Comment:
can we make the classes in this module directory like so

```
/ selector
      StorageSelector
      BaseStorageSelector
      /impl
              DefaultStorageSelector
              StorageInNameSelector
      
```

Reply:
done